depends/packages/qt.mk

```diff
@@ -51,6 +51,7 @@ $(package)_config_opts += -no-linuxfb
 $(package)_config_opts += -no-libjpeg
 $(package)_config_opts += -no-libproxy
 $(package)_config_opts += -no-libudev
+$(package)_config_opts += -no-mimetype-database
```

Comment:
> @kittywhiskers
[merge](https://github.com/dashpay/dash/pull/5355/commits/5c8be53169bc8a2abe051fa3e0a958711fd7d326) https://github.com/bitcoin/bitcoin/pull/26057[: Get rid of perl dependency](https://github.com/dashpay/dash/pull/5355/commits/5c8be53169bc8a2abe051fa3e0a958711fd7d326)

This backports can't be included yet, because option `-no-mime-database` is available since Qt 5.13.0, but our version is 5.12.11